hack/update-gomod

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
is this standard for csi drivers to manage deps like this?

Reply:
Yep - here are some examples from the NFS CSI driver: https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/hack/update-gomod.sh
and the FSxL CSI driver: https://github.com/kubernetes-sigs/aws-fsx-csi-driver/blob/master/hack/update-gomod